src/alert/lsst.rs

```diff
@@ -413,9 +435,25 @@ pub struct DiaNondetectionLimit {
     pub band: String,
     #[serde(rename = "diaNoise")]
     pub dia_noise: f32,
+}
 
-    // these fields are computed later, from the dia_noise
-    pub diffmaglim: Option<f32>,
+#[derive(Debug, PartialEq, Clone, serde::Deserialize, serde::Serialize)]
+pub struct NonDetection {
+    #[serde(flatten)]
+    pub dia_nondetection_limit: DiaNondetectionLimit,
+    pub diffmaglim: f32,
+}
+
+impl TryFrom<DiaNondetectionLimit> for NonDetection {
```

Comment:
This one can just be `impl From` because the conversion never results in an error.

Reply:
done